_pages/dev/performance/above-the-fold.md

```diff
@@ -8,21 +8,21 @@ title: Above-the-Fold Loading (DRAFT)
 
 {% include docs/feature_version.html content=version_note %}
 
-Above-the-fold loading is a technique to prioritize the creation of components which are above the fold. Above-the-fold is traditionally known as the upper half of a newspaper, where the most important stories are located. This transfers to the web as all components which are placed at the top of the page, where the experience starts.
+Above-the-fold loading is a technique that prioritizes the creation of components that are "above the fold". The term "above the fold" is traditionally known as the upper-half of a newspaper, where the most important stories are located. When this concept is transferred to the web, it refers to all the components that are placed at the top of the page, where the experience starts.
 
-Above-the-fold loading requires two important ingredients:
+Above-the-fold loading requires the following important ingredients:
 
-1. Deferred loading, which is a technique to postpone the creation of components _below_ the fold. For more information, see [Deferred Loading]({{ site.baseurl }}{% link _pages/dev/performance/deferred-loading.md %}).
-2. The notion of the _page fold_. The page fold is not static, and differs from device to device, screen to screen and even the size of the browser.
+1. Deferred loading, which is a technique that postpones the creation of components that are "below the fold". For more information, see [Deferred Loading]({{ site.baseurl }}{% link _pages/dev/performance/deferred-loading.md %}).
+2. The notion of the "page fold". The page fold is not static, and differs from device to device, from screen to screen, and even changes depending on the size of the browser.
 3. A couple of CSS rules to control the page fold.
 
 ## Page Fold Configuration
 
-The page fold is configurable per page template and breakpoint. The page fold configuration is only an indication to speed up the initial creation of page slots above the fold. Page slots are eventually rendered if they're above the fold. The indicated page slot however will be used to prioritize all above the fold page slots over page slots below the fold.
+The page fold is configurable for each page template and breakpoint. The page fold configuration is only an indication to speed up the initial creation of page slots that are above the fold. All page slots are eventually rendered if they are above the fold. You designate the page fold by assigning a page slot to the `pageFold` parameter. This page slot, and all page slots above it, are "above the fold". These page slots are prioritized ahead of page slots that are "below the fold".
```

Comment:
> All page slots are eventually rendered if they are above the fold

Can we rephrase this to something like:
"All page slots are eventually rendered if they happen to be _above_ the fold"

Reply:
> This page slot, and all page slots above it

what about: 
"This page slot, and all previous sibling page slots, ..."

Reply:
Done.